src/resolvers-cassandra/Tiles/queries.js

```diff
@@ -1,40 +1,252 @@
 'use strict';
 
+const Promise = require('promise');
+const cassandraConnector = require('../../clients/cassandra/CassandraConnector');
+const featureServiceClient = require('../../clients/locations/FeatureServiceClient');
 const withRunTime = require('../shared').withRunTime;
 
 /**
+ * @see http://wiki.openstreetmap.org/wiki/Slippy_map_tilenames#Python
+ */
+function deg2num(lat_deg, lon_deg, zoom) {
```

Comment:
I believe this same functionality is already available in `Tile.tileIdFromLatLong` within the `geotile` module